### DISCUSSION TITLE
ADE now requires Log-In

### DISCUSSION THREAD

**Discussion by SlickRick1987** [SlickRick1987/AdultDVDEmpire.bundle#5] [General]
The old code and systems don't work anymore as ADE has added a required log-in.  I have rebuild the system for my personal use, but it is technical and convoluted as it requires the use of an accompanying Proxy Server that needs to run on Python outside of Plex and I have updated the Plex Agent to work with the Proxy Server.  Hesitant to post this solution as it remains a bit wonky, but is getting the job done for me.  If anyone out there still cares and wants to see the workaround, let me know.  If no one asks, I will assume everyone else has moved on.

**Comment by theoneandonlykinge** [comment]
i still care! is there a way i can give in my userrname password to make it work?

**Reply by SlickRick1987** [reply]
Sorry for delay, ADE keeps changing their systems, so for a while there were nearly weekly updates to the code to try to keep up.  But ADE has been stable for two weeks now.  Maybe they are finally happy with their new system.  Like I said above, the work around is not straight forward, as the update by-passes the native Plex limitations by coordinating with python scripting and a proxy server running outside of Plex.  I just kick-off a python script in PowerShell before running updates in Plex and the scripting handles the rest,  However you have to be comfortable with python and loading a few applications for the proxy server that python can utilize.  If you are game to give it a try, I will update the README to be as clear as I can and post the code.

**Comment by theoneandonlykinge** [comment]
Thank you so much for offering to share your solution and for the time you’ve invested in this project. It’s clear you’ve put a lot of effort into making everything run smoothly. Unfortunately, since I don’t know how to program, I’m afraid I won’t be able to implement the workaround you’ve described. While it sounds like a fantastic system, I’ll have to continue handling things manually for now. I really appreciate your willingness to help and the care you’ve put into this project. Best of luck with everything, and thank you again for your time and support!





 

Van: SlickRick1987 ***@***.***> 
Verzonden: donderdag 5 september 2024 15:24
Aan: SlickRick1987/AdultDVDEmpire.bundle ***@***.***>
CC: theoneandonlykinge ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [SlickRick1987/AdultDVDEmpire.bundle] ADE now requires Log-In (Discussion #5)

 

Sorry for delay, ADE keeps changing their systems, so for a while there were nearly weekly updates to the code to try to keep up. But ADE has been stable for two weeks now. Maybe they are finally happy with their new system. Like I said above, the work around is not straight forward, as the update by-passes the native Plex limitations by coordinating with python scripting and a proxy server running outside of Plex. I just kick-off a python script in PowerShell before running updates in Plex and the scripting handles the rest, However you have to be comfortable with python and loading a few applications for the proxy server that python can utilize. If you are game to give it a try, I will update the README to be as clear as I can and post the code.

**Reply by SlickRick1987** [reply]
Let me give some thought to how I might make the process easier.  Will keep you posted.

**Comment by theoneandonlykinge** [comment]
Big thanks!!!

 

Van: SlickRick1987 ***@***.***> 
Verzonden: donderdag 12 september 2024 00:59
Aan: SlickRick1987/AdultDVDEmpire.bundle ***@***.***>
CC: theoneandonlykinge ***@***.***>; Comment ***@***.***>
Onderwerp: Re: [SlickRick1987/AdultDVDEmpire.bundle] ADE now requires Log-In (Discussion #5)

 

Let me give some thought to how I might make the process easier. Will keep you posted.
